src/tests/shamrock/solvergraph/IDataEdgeSerializable_tests.cpp

```
@@ -7,38 +7,38 @@
+        REQUIRE_EQUAL(restored.type_name(), "IDataEdgeSerializable<f64>");
     }
 
     {
         auto ptr      = shamrock::solvergraph::JsonSerializable::from_json(j);
         auto *as_edge = dynamic_cast<Edge *>(ptr.get());
 
         REQUIRE(as_edge != nullptr);
-        REQUIRE_EQUAL(as_edge->value, original.value);
+        REQUIRE_EQUAL(as_edge->data, original.data);
         REQUIRE_EQUAL(as_edge->get_label(), "my_label");
         REQUIRE_EQUAL(as_edge->get_raw_tex_symbol(), "my_tex");
```

Cppcheck (2.21.0)

[performance] Function parameter 's' should be passed by const reference.

(passedByValue)